handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,51 @@
+   2. We should also audit the logs that we currently ingest to ensure they contain sufficient information. If they don't, we need to increase their scope.
+7. We're ingesting logs from all services loosely coupled to our Cloud deployment. This is anything that cannot independently impact Cloud, but may be leveraged to gain access, extract sensitive information, or otherwise negatively impact cloud.
+   1. For example, non-production GCP projects, Slack, honeycomb, Zoom, etc.
+8. Normalize log format in our centralized log storage so that it is easier to correlate and search. This is a prerequisite for creating automated alerts from the logs.
+9. Create alerts and dashboards to automate the process of investigating events of interest (e.g., detect and alert on a spike of failed login attempts to a single account, or across Sourcegraph Cloud as a whole).
+10. We run a time-bound capture the flag event where there are larger bounties for being able to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+11. We connect our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) to Sourcegraph Cloud and only [members who can access that repository on GitHub](https://github.com/sourcegraph/security-test/settings/access) can access that repository on Sourcegraph Cloud (i.e., [Sourcegraph organization owners](https://github.com/orgs/sourcegraph/people?query=role%3Aowner) and [@sourcegraph/security](https://github.com/orgs/sourcegraph/teams/security) members). We then advertise a bounty for each unique vulnerability that allows an unauthorized person to gain access to this [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+12. We host our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph Cloud.
+13. Document and publish our security practices so that our customers can 
+14. Document our roadmap and progress towards [SOC 3](#soc-audit-readiness).
+15. Validate that we are GDPR compliant.
+16. **Sourcegraph Cloud is a viable alternative to Sourcegraph for SMBs**
+
+
+
+### SOC Audit readiness
+
+**Problem:** As we begin providing a SaaS product, attaining a full SOC 2/3 certification will become necessary to enable the business. Note that a SOC 3 provides the same assurances as a SOC 2 certification, but contains less details, which makes it more accessible to general users, and something that we can [freely distribute](https://www.aicpa.org/interestareas/frc/assuranceadvisoryservices/aicpasoc3report.html). Therefore, we intend to pursue a SOC 3 certification in keeping with our company values of [openness and transparency](../../../company/values.md#open-and-transparent). If reasonable, we'll pursue SOC 2 type 2 in parallel with this, since it's the same audit process.
+
+1. Document our roadmap and progress towards SOC 3.
```

Comment:
This presents some duplication with what Vanta does. As they reduce this specific pain point with their ongoing GAP analysis, it could use some rethinking.

Reply:
Reworded to note that we're going to configure vanta to document/plan SOC readiness.